### PR TITLE
(nssm) migrated from dtgm; embedded, modernized.

### LINKED ISSUE #759: (nssm) migrate from dtgm
**The Non-Sucking Service Manager**

* Package: https://chocolatey.org/packages/NSSM
* Should be embedded, public license
* Outdated ATM.

### PR DESCRIPTION
**Pull request by majkinetor**
Closes #759

### CODE REVIEW

**Comment by majkinetor** [comment]
Ping @AdmiringWorm

**Comment by AdmiringWorm** [comment]
@majkinetor I've not forgotten about this one, I've had my plate quite full lately so haven't had the time to review this PR yet.
As well as the long moderation queue we had until yesterday (spent pretty much all sunday on it).

I'll try reviewing this PR at some point today, but can't right now.

**Comment by majkinetor** [comment]
I didn't requested review, its trivial embedded package. Just add choco user. Its certainly better then what is currently on.

Lets not official review hinder the process as you can always rise the issue later if needed.

File: automatic/nssm/nssm.nuspec [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+</description>
+    <summary>Service helper program similar to srvany and cygrunsrv</summary>
+    <releaseNotes>http://nssm.cc/changelog</releaseNotes>
+    <copyright>© Iain Patterson</copyright>
+    <tags>service foss admin cli</tags>
+    <projectSourceUrl>http://git.nssm.cc/?p=nssm.git</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/nssm</packageSourceUrl>
+    <docsUrl>http://nssm.cc/commands</docsUrl>
+  </metadata>
+</package>
```
Comment (AdmiringWorm) [inline automatic/nssm/nssm.nuspec]: Missing `<files>` element

File: automatic/nssm/legal/VERIFICATION.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+VERIFICATION
+
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+Package can be verified like this:
+
```
Comment (AdmiringWorm) [inline automatic/nssm/legal/VERIFICATION.txt]: The location where the binary file is located should also be inside the VERIFICATION.txt file (ie `http://www.nssm.cc/download`), and not just the full URL to the binary file.

File: automatic/nssm/update.ps1 [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+function global:au_AfterUpdate  { Set-DescriptionFromReadme -SkipFirst 2 }
+
+function global:au_GetLatest {
+    $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
+    $re    = 'release.+\.zip$'
+    $url   = $download_page.links | ? href -match $re | select -First 1 -expand href
+
+    @{
+        Version = $url -split '-|\.zip' | select -Last 1 -Skip 1
+        URL32   = "http://www.nssm.cc/$url"
```
Comment (AdmiringWorm) [inline automatic/nssm/update.ps1]: Should probably use `https`, unless I'm mistaken I believe they support https.

**Comment by majkinetor** [comment]
Done

**Comment by majkinetor** [comment]
> Should still wait on the merge though, until the package have been removed from dtgm's repository.

You know as well as I do that we can't rely on that, dtgm's involvement is nowdays less then minimal.

**Pull request merged**